### PR TITLE
InfluxDB: support flux editor for template queries

### PR DESCRIPTION
**Pull request by ryantxu**
This removes the annotation work from #26634 and only solves template variables.  I will try to refactor that so it is more general/reusable behavior

This is the equivalent of "SHOW MEASUREMENTS" using flux:
image

Should help: #26780

### CODE REVIEW

File: public/app/plugins/datasource/influxdb/components/FluxQueryEditor.tsx [hunk 1]
```diff
@@ -60,7 +60,7 @@ v1.measurements(bucket: v.bucket)`,
     label: 'Schema Exploration: (fields)',
     description: 'Return every possible key in a single table',
     value: `from(bucket: v.bucket)
-  |> range(start: v.timeRangeStart, stop:timeRangeStop)
+  |> range(start: v.timeRangeStart, stop:v.timeRangeStop)
```
Comment (ryantxu) [inline public/app/plugins/datasource/influxdb/components/FluxQueryEditor.tsx:63]: this was a typo

**Review by srclosson — APPROVED** [review]
LGTM

**Review by 5hsil — APPROVED** [review]
5hisl

**Review by 5hsil — COMMENTED** [review]
27370

**Pull request merged**